src/main/kotlin/no/nav/klage/oppgave/api/OppgaveController.kt

```diff
@@ -22,6 +26,23 @@ class OppgaveController(val oppgaveService: OppgaveService, val saksbehandlerSer
         private val logger = getLogger(javaClass.enclosingClass)
     }
 
+    @ApiOperation(value = "Hent oppgaver", notes = "Henter alle oppgaver som saksbehandler har tilgang til.")
+    @GetMapping("/ansatte/{navIdent}/ikketildelteoppgaver", produces = ["application/json"])
+    fun getOppgaver(
+        @ApiParam(value = "NavIdent til en ansatt")
+        @PathVariable navIdent: String,
+        @ApiParam(value = "Klage eller anke")
+        @RequestParam(name = "type", required = false) type: String,
+        @RequestParam(required = false) ytelse: String,
```

Comment:
Jeg antar at jeg kunne googlet meg til dette, men hva gjør Springfox når man har en RequestParam uten å ha en ApiParam?

Reply:
ApiParam er bare for å gi mer info enn hva som kan utledes. SpringFox vil vise info uansett.